support/SubmitScripts/Wheeler.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -
+#SBATCH -o spectre.out
+#SBATCH -e spectre.out
+#SBATCH --ntasks-per-node 24
+#SBATCH -A sxs
+#SBATCH --no-requeue
+#SBATCH -J my_job_name
+#SBATCH --nodes 4
+#SBATCH -t 0:15:00
+
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+# To run a job on Wheeler:
+# - Set the job name, number of nodes, and time limit above.
+# - Set the build directory, run directory, executable name,
+#   and input file. The input file path is relative to ${RUN_DIR}.
+#
+# Optionally, if you need more control over how SpECTRE is launched on
+# Wheeler you can edit the launch command at the end of this file directly.
+#
+# To submit the script to the queue run:
+#   sbatch Wheeler.sh
+
+export SPECTRE_BUILD_DIR=/path/to/build/dir
+export RUN_DIR=/panfs/ds08/sxs/run/dir
+export SPECTRE_EXECUTABLE=EvolveScalarWave3D
+export SPECTRE_INPUT_FILE=./Input3DPeriodic.yaml
+
+umask 0022
+export PATH=${SPECTRE_BUILD_DIR}/bin:$PATH
```

Comment:
Do we want to do the SpEC thing and copy the executable to avoid the job breaking if SpECTRE is recompiled before it runs?  That would have to be done before this script runs, though.

Reply:
I'm not sure, and that might only be determined once we figure out how to replace BFI. I'm not sure we want something like `DoMultipleRuns` since that tries to partially emulate things like [Pegasus](https://pegasus.isi.edu/overview/), which would generally be preferable for managing very large number of jobs. When I first started using SpEC I found all the automation around job submitting to actually be a hindrance since there was no documentation on it but I was familiar with TORQUE and batch systems in general.

Another complication will be what we do with segmenting runs, which is not worked out yet.

My personal preference for right now is to let people manually manage these things for the time being and then figure out what ways there are to automate some things.

Reply:
If we want people to manage this on their own the instructions here should tell them that.  People coming from SpEC will not expect their runs to break if they touch their build directory.

I don't think batch systems will be able to solve this problem.  I don't know anything about Pegasus, and it has too much documentation for me to digest at the moment.

Reply:
I don't really know where to say this in the documentation.

Reply:
I guess in this file is one place, though I wonder if that would be enough/best. I don't think we currently have a better location, though.

Reply:
The note looks good.